app/components/ui/badge.tsx

```diff
@@ -9,7 +9,7 @@ const badgeVariants = cva(
 		variants: {
 			variant: {
 				default: "border-white bg-primary shadow-sm hover:bg-primary/80 rounded-xl",
-				secondary: "border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80",
```

Comment:
hmm, shouldn't text-secondary-foreground be compatible with bg-secondary ?

Reply:
Moved it back.